### PR TITLE
Update: Czech translation up to 0.54.4.

### PR DESCRIPTION
**Pull request by JakMel**
Tried to update the Czech translation. I am sorry for my bad git skills, I just love my language and I'm not a Linux user, I hope this will work. Please be merciful ':)

### CODE REVIEW

File: src/lang/extra/czech.txt [hunk 1]
```diff
@@ -572,81 +1106,1083 @@ STR_PROGSIG_INSERT                                              :Vložit
...
+STR_BREAKDOWN_TYPE_LOW_SPEED                                    :omezení na {VELOCITY}
+STR_BREAKDOWN_TYPE_LOW_POWER                                    :{COMMA}% výkon
+STR_BREAKDOWN_TYPE_HIT_RV                                       :srážka se silničním vozidlem
+STR_BREAKDOWN_TYPE_BRAKES_OVERHEATED                            :přehřáté brzdy
+STR_BREAKDOWN_TYPE_DEPOT                                        :Míří do hangáru na {STATION} pro provedení oprav
+STR_BREAKDOWN_TYPE_LANDING                                      :Míří na {STATION} pro nouzové přistání
+
+STR_ERROR_TRAIN_TOO_HEAVY                                       :{WHITE}Souprava {VEHICLE} je příliš těžká
+
+STR_VEHICLE_STATUS_HEADING_FOR_DEPOT_SELL_VEL                   :{CREAM}Míří na {PUSH_COLOUR}{RED}prodej{POP_COLOUR} do {DEPOT}, {VELOCITY}# Cargo type orders Window
```
Comment (JGRennison) [inline src/lang/extra/czech.txt:1498]: It seems there is some stray text here

File: src/lang/extra/czech.txt [hunk 2]
```diff
@@ -572,81 +1106,1083 @@ STR_PROGSIG_INSERT                                              :Vložit
...
+STR_VEHICLE_LIST_CARGO_LIST                                     :{STRING}, náklad: {CARGO_LIST}
+STR_VEHICLE_LIST_RELIABILITY                                    :{STRING}, spolehlivost: {COMMA}%
+STR_VEHICLE_LIST_RELIABILITY_RED                                :{STRING}, spolehlivost: {RED}{COMMA}%
+STR_VEHICLE_LIST_MAX_SPEED                                      :{STRING}, max. rychlost: {VELOCITY}
+STR_VEHICLE_LIST_ENGINE_BUILT                                   :{STRING}, řada: {ENGINE}, rok výroby: {NUM}
+STR_VEHICLE_LIST_VALUE                                          :{STRING}, hodnota: {CURRENCY_LONG}
+STR_VEHICLE_LIST_LENGTH                                         :{STRING}, délka: {DECIMAL} políč{P ko ka ek}
+STR_VEHICLE_LIST_TIME_TO_LIVE                                   :{STRING}, životnost: {COMMA} {P rok roky let} zbývá
+STR_VEHICLE_LIST_TIME_TO_LIVE_RED                               :{STRING}, životnost: {RED}{COMMA} {BLACK}{P rok roky let} zbývá
+STR_VEHICLE_LIST_TIME_TO_LIVE_OVERDUE                           :{STRING}, životnost: {RED}{COMMA} {BLACK}year{P "" s} overdue
```
Comment (JGRennison) [inline src/lang/extra/czech.txt:1425]: Part of the text does not seem to be translated?

Comment (JakMel) [reply src/lang/extra/czech.txt]: Oh, yes, sorry. In addition, there was a wrong translation of the next string. Both fixed

File: src/lang/extra/czech.txt (hunk 1 above)
Comment (JakMel) [reply src/lang/extra/czech.txt]: Oh, yes, sorry. In addition, there was a wrong translation of the next string. Both fixed

File: src/lang/extra/czech.txt (hunk 2 above)
Comment (JakMel) [reply src/lang/extra/czech.txt]: Sorry for that. Fixed

**Comment by JGRennison** [comment]
Thanks for this

**Pull request merged**